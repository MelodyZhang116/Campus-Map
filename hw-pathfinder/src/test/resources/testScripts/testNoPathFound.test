# test finding the path which does not exist

CreateGraph g

AddNode g n1
AddNode g n2
AddNode g n3
AddEdge g n1 n2 100

FindPath g n1 n2

FindPath g n1 n3