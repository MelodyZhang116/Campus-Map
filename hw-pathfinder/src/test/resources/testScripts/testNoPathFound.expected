# test finding the path which does not exist

created graph g

added node n1 to g
added node n2 to g
added node n3 to g
added edge 100.000 from n1 to n2 in g

path from n1 to n2:
n1 to n2 with weight 100.000
total cost: 100.000

path from n1 to n3:
no path found